docs/dev/generator.js

```diff
@@ -536,6 +552,22 @@ const processMd = async ({baseConfig, relDir, in_folder, iPath, oPath}) => {
   theader.remove();
 
   // Process code snippets.
+  const linkifySpans = (elem, language) => {
+    const linkifySpan = (s, language) => {
+      const text = s.textContent
+      const ref = xrefGetMaybe(language == "reach" ? "rsh" : language, text);
```

Comment:
I don't think this `?:` is needed because of the normalization

Reply:
Oh, yeah.